spec/fixtures/graphql_schema.rb

```diff
@@ -29,7 +30,7 @@
     query QueryType
     use BatchLoader::GraphQL
   end
-when '1_8'
+elsif
```

Comment:
In case there's any destructive changes with a future version of graphql-ruby, we will be able to add new conditional branch here (we may want to update our README as well)